openmc/surface.py

```diff
@@ -282,36 +279,43 @@ class XPlane(Plane):
 
     Parameters
     ----------
-    surface_id : int
+    surface_id : int, optional
         Unique identifier for the surface. If not specified, an identifier will
         automatically be assigned.
-    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}, optional
         Boundary condition that defines the behavior for particles hitting the
         surface. Defaults to transmissive boundary condition where particles
         freely pass through the surface.
-    x0 : float
-        Location of the plane
-    name : str
+    x0 : float, optional
+        Location of the plane. Defaults to 0.
+    name : str, optional
         Name of the plane. If not specified, the name will be the empty string.
 
     Attributes
     ----------
     x0 : float
         Location of the plane
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+        Boundary condition that defines the behavior for particles hitting the
```

Comment:
Just curious, is the notation that a parameter docstring which wraps across two lines ends with a period, but those which finish in a single line do not require a period?